core/src/main/java/io/questdb/cairo/wal/seq/TransactionLogCursor.java

```diff
@@ -51,4 +51,10 @@ public interface TransactionLogCursor extends Closeable {
     void setPosition(long txn);
 
     void toTop();
+
+    long getPartNo();
+
+    // Sets cursor to minimum available position.
```

Comment:
terminology, it might be easier to use `txn` instead, such as min txn, vs minimum position etc

Reply:
renamed to `toMinTxn()`